scudcloud-1.0/lib/scudcloud.py

```diff
@@ -224,6 +230,8 @@ def createAction(self, text, slot, shortcut=None, checkable=False):
         if shortcut is not None:
             action.setShortcut(shortcut)
         action.triggered.connect(slot)
+        if shortcut is not None:
+            self.addAction(action)
```

Comment:
I wasn't sure about interactions between action.triggered.connect and self.AddAction, but I suppose either way I could have just move the connect call up.